src/test/java/org/jenkinsci/plugins/workflow/cps/DSLTest.java

```diff
@@ -33,6 +33,7 @@
 import java.util.Set;
 import static org.hamcrest.Matchers.containsString;
 
+import org.hamcrest.Matchers;
```

Comment:
This import appears to be unused as of 8af1b4aa7501e3fcca039ffb5c25861eba0aa86a and can now be deleted.